src/parser/estree_translator.ml

```diff
@@ -966,6 +966,7 @@ end with type t = Impl.t) = struct
       | Literal.Boolean b -> bool b
       | Literal.Null -> null
       | Literal.Number f -> number f
+      | Literal.BigInt f -> number f
```

Comment:
So I looked a little into estree and it looks like they're working on how to represent bigints in the AST (https://github.com/estree/estree/pull/179).

One downside, however, is that the spec seems to want to set `value` to be a bigint, and I'm not sure if js_of_ocaml can support that.

Reply:
JSON doesn't support it either

Reply:
https://github.com/estree/estree/pull/179#discussion_r158063172 `value` property also supports null, so translator can just return null